draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -1901,9 +1904,66 @@ and feedback.
 All references to {{RFC4122}} in the IANA registries should be replaced with references to this document.
 References to {{RFC4122}} document's Section 4.1.2 should be updated to refer to this document's {{format}}.
 
-There is no update required to the IANA URN namespace registration {{URNNamespaces}} for UUID filed in {{RFC4122}}.
-
-IANA is not required to track UUIDs used for identifying items such as versions, variants, namespaces, or hashspaces.
+The IANA URN namespace registration {{URNNamespaces}} for UUID filed in {{RFC4122}} should be updated to reference this document.
+
+Finally IANA should track UUID Variants, UUID Subtypes, and Special Case "Reserved UUIDs" such as Min, Max, and Namespace IDs as specified in {{iana1}}, {{iana2}}, {{iana3}}.
+
+The policy for these three registries is "Specification Required", per {{RFC8126}}, Section 4.6.
+
+When evaluating requests, the designated expert(s) should consider community feedback, how well-defined is the reference specification, and this specification's requirements.
+Vendor-specific, application-specific, and deployment-specific values are unable to be registered.
+Specification documents should be published in a stable, freely available manner (ideally located with a URL) but need not be standards.
+The designated experts will either approve or deny the registration request, and communicate their decision to IANA. Denials should include an explanation and, if applicable, suggestions as to how to make the request successful.
+
+## IANA UUID Variants Registry and Registration {#iana1}
+This specification defines the "UUID Variants" registry for common, widely used UUID standards.
+
+{{ianaVariants}} should be used as-is for this registry with "This document" replaced as required.
+
+| Name           | Bits | Variant | Reference                  |
```

Comment:
Sure, easy enough change although nothing technically out there in real world uses more than 3 var bits at the moment right? 
It can be modified later, I was just going off of what we have in the table of _this_ document.

Reply:
Family is 8-bit on the same byte-location. If we parse the variant also like a 8-bit, it is easy to convert it to an family integer: `if(variant<0x80){ family = variant & 0b1111111; }`. It also gives us the possibility to assign numbers (or number ranges) to a variant, which is not working for variable-size bit maps.

Reply:
Hmm, okay I see where you are going. 
I am going to leave it for the moment based on what we have in this doc. 
We can flyways update the IANA registration later down the line in the historical RFC when we flesh out family. 
A side topic, I don't want to derail the thread so I am going to resolve this one for the moment.